lib/puppet/provider/netapp_security_login/cmode.rb

```diff
@@ -59,7 +59,7 @@ def flush
     when :present
       #if lock status changes, do the right thing
       if @original_values[:is_locked] != is_locked
-        if is_locked
+        if is_locked == "true"
```

Comment:
We should have an `== true` check here too it seems like, because typing in puppet is hard.